app/assets/javascripts/discourse/app/lib/plugin-api.gjs

```diff
@@ -883,6 +883,7 @@ class PluginApi {
       {
         since: "v3.4.0.beta3-dev",
         id: "discourse.post-menu-widget-overrides",
+        url: "https://meta.discourse.org/t/upcoming-post-menu-changes-how-to-prepare-themes-and-plugins/341014",
```

Comment:
Tiny nitpick: I think deprecation messages are more readable if we use the short version of the topic URL like `https://meta.discourse.org/t/341014`

Also, might be nice to move the deprecation info into a constant, like


```
const POST_MENU_DEPRECATION_INFO = {
  since: ...,
  id: ...,
  url: ...
}
```
and then reference it, rather than duplicating the info 4 times in the same file.

Reply:
Done